### DISCUSSION TITLE
coco joint_weights

### DISCUSSION THREAD

**Discussion by smidm** [open-mmlab/mmpose#2834] [Q&A]
Most datasets use equal weights for all joints in training loss. Only [COCO](https://github.com/open-mmlab/mmpose/blob/6d10b2ec81da7e252016b3154c7fdb46c403ecd8/configs/_base_/datasets/coco.py#L174) and few related datasets use following:
```python
joint_weights=[
  1., 1., 1., 1., 1., 1., 1.,  # 'nose', 'left_eye', 'right_eye', 'left_ear', 'right_ear', 'left_shoulder', 'right_shoulder',   
  1.2, 1.2,  # 'left_elbow', 'right_elbow',   
  1.5, 1.5,  # 'left_wrist', 'right_wrist', 
  1., 1.,    # 'left_hip', 'right_hip',
  1.2, 1.2,  # 'left_knee', 'right_knee', 
  1.5, 1.5,  # 'left_ankle', 'right_ankle',
]
```
Is there a source or a reason for these particular weights?

**Comment by Ben-Louis** [comment]
We have transfered these weights from [HRNet](https://github.com/HRNet/HRNet-Human-Pose-Estimation/blob/00d7bf72f56382165e504b10ff0dddb82dca6fd2/lib/dataset/coco.py#L97-L101), but they are rarely used.
